pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -58,6 +59,34 @@ type ClientCache struct {
 	// argument evaluation
 	discoveryClientFactory DiscoveryClientFactory
 	discoveryClient        discovery.DiscoveryInterface
+
+	kubernetesClientLoader KubernetesClientCache
+}
+
+// kubernetesClientLoader provides a new kubernetes.Clientset
+type KubernetesClientCache struct {
+	// once makes sure the client is only initialized once
```

Comment:
Ensure the client is only initialized once or the cache is only initialized once?